From 7acdbbcf080d4f9c06dcf8912ccf4d1b2184f97a Mon Sep 17 00:00:00 2001
From: Stuart Archibald <redacted>
Date: Fri, 22 Feb 2019 10:44:00 +0000
Subject: [PATCH 07/15] define mkl_version in __init__.py

---
 numpy/__init__.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/numpy/__init__.py b/numpy/__init__.py
index ba88c73..3723daa 100644
--- a/numpy/__init__.py
+++ b/numpy/__init__.py
@@ -192,6 +192,8 @@ else:
     del PytestTester
 
 
+    __mkl_version__ = "__MKL_VERSION__"
+
     def _sanity_check():
         """
         Quick sanity checks for common bugs caused by environment.
-- 
2.20.1

