diff --git a/srclib/stdio.in.h b/srclib/stdio.in.h
index d706377..72ff11a 100644
--- a/srclib/stdio.in.h
+++ b/srclib/stdio.in.h
@@ -743,7 +743,7 @@ _GL_WARN_ON_USE (getline, "getline is unportable - "
    so any use of gets warrants an unconditional warning; besides, C11
    removed it.  */
 #undef gets
-#if HAVE_RAW_DECL_GETS && !defined __cplusplus
+#if defined(__GLIBC__) && !defined(__UCLIBC__) && (!defined(__GLIBC_PREREQ) || !__GLIBC_PREREQ(2, 16)) && HAVE_RAW_DECL_GETS && !defined __cplusplus
 _GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 #endif
 
