From ac53a3d1bc90b2e8c6163782cc1ca8daa468ae93 Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Thu, 26 Jul 2018 11:30:25 -0500
Subject: [PATCH 07/15] define mkl_version in __init__.py

---
 numpy/__init__.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/numpy/__init__.py b/numpy/__init__.py
index 1f60f07..144569d 100644
--- a/numpy/__init__.py
+++ b/numpy/__init__.py
@@ -201,6 +201,8 @@ else:
     del PytestTester
 
 
+    __mkl_version__ = "__MKL_VERSION__"
+
     def _sanity_check():
         """
         Quick sanity checks for common bugs caused by environment.
-- 
2.7.4

