From 46c65ff19ff2ea5bbd41aa875733734e00056313 Mon Sep 17 00:00:00 2001
From: Stuart Archibald <redacted>
Date: Fri, 22 Feb 2019 11:10:36 +0000
Subject: [PATCH 12/15] Remove ICC specific flags

---
 numpy/core/setup.py | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/numpy/core/setup.py b/numpy/core/setup.py
index 486319e..6e7b64a 100644
--- a/numpy/core/setup.py
+++ b/numpy/core/setup.py
@@ -933,12 +933,13 @@ def configuration(parent_package='',top_path=None):
             join(codegen_dir, 'generate_ufunc_api.py'),
             ]
 
-    if platform.system() == "Windows":
-        eca = ['/fp:fast=2', '/Qimf-precision=high', '/Qprec-sqrt', '/Qstd=c99']
-        if sys.version_info < (3, 0):
-            eca.append('/Qprec-div')
-    else:
-        eca = ['-fp-model', 'fast=2', '-fimf-precision=high', '-prec-sqrt']
+#    if platform.system() == "Windows":
+#        eca = ['/fp:fast=2', '/Qimf-precision=high', '/Qprec-sqrt', '/Qstd=c99']
+#        if sys.version_info < (3, 0):
+#            eca.append('/Qprec-div')
+#    else:
+#        eca = ['-fp-model', 'fast=2', '-fimf-precision=high', '-prec-sqrt']
+    eca=[]
     config.add_library('loops',
                        sources=loops_src,
                        include_dirs=[],
@@ -958,7 +959,7 @@ def configuration(parent_package='',top_path=None):
                                   generate_umath_c,
                                   generate_ufunc_api,
                                  ] + aligned_alloc_sources,
-                         extra_compile_args=['/Qstd=c99' if platform.system() == "Windows" else ''],
+                         extra_compile_args=[],
                          depends=deps + multiarray_deps + umath_deps +
                                 common_deps,
                          libraries=['loops', 'npymath', 'npysort', 'mkl_cp'],
-- 
2.20.1

