From 83c87b62d7521a4c06bf50bd2a81185da78825c1 Mon Sep 17 00:00:00 2001
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Thu, 28 Feb 2019 16:57:00 -0600
Subject: [PATCH 16/17] define __THRESHOLD

---
 numpy/core/src/multiarray/item_selection.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/numpy/core/src/multiarray/item_selection.c b/numpy/core/src/multiarray/item_selection.c
index ae144f2..b857bc9 100644
--- a/numpy/core/src/multiarray/item_selection.c
+++ b/numpy/core/src/multiarray/item_selection.c
@@ -31,6 +31,10 @@
 
 #define memmove(src, dst, size) call_mkl_mv(src, dst, size, __FILE__, __func__, __LINE__)
 
+#ifndef __THRESHOLD
+#define __THRESHOLD 524288
+#endif
+
 /*NUMPY_API
  * Take
  */
-- 
2.7.4

