From 727131871fded9d765a81d4fd79b826a8526654a Mon Sep 17 00:00:00 2001
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Thu, 28 Feb 2019 17:46:30 -0600
Subject: [PATCH 17/17] mark known failing tests on ppc64le

---
 numpy/core/tests/test_umath.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/numpy/core/tests/test_umath.py b/numpy/core/tests/test_umath.py
index 2f8edeb..2972b69 100644
--- a/numpy/core/tests/test_umath.py
+++ b/numpy/core/tests/test_umath.py
@@ -2465,6 +2465,8 @@ class TestComplexFunctions(object):
             assert_almost_equal(fz.real, fr, err_msg='real part %s' % f)
             assert_almost_equal(fz.imag, 0., err_msg='imag part %s' % f)
 
+    @pytest.mark.xfail(condition=platform.machine().startswith("ppc64"),
+                       reason="poor precision on older ppc64le glibc")
     def test_precisions_consistent(self):
         z = 1 + 1j
         for f in self.funcs:
@@ -2544,6 +2546,8 @@ class TestComplexFunctions(object):
                 assert_(abs(a - b) < atol, "%s %s: %s; cmath: %s" % (fname, p, a, b))
 
     @pytest.mark.parametrize('dtype', [np.complex64, np.complex_, np.longcomplex])
+    @pytest.mark.xfail(condition=platform.machine().startswith("ppc64"),
+                       reason="poor precision on older ppc64le glibc")
     def test_loss_of_precision(self, dtype):
         """Check loss of precision in complex arc* functions"""
 
-- 
2.7.4

