From 98f3c78372160787c1945eeeedcf1e716fc2c937 Mon Sep 17 00:00:00 2001
From: "Jonathan J. Helmus" <jjhelmus@gmail.com>
Date: Thu, 1 Feb 2018 13:37:27 -0600
Subject: [PATCH] disable msys compiler check

---
 configure | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 6d37d65..193f030 100755
--- a/configure
+++ b/configure
@@ -135,8 +135,8 @@ case $(uname -s) in
 esac
 
 # Search the PATH and pick the first match.
-findTool "Cygwin/MSYS G++ compiler" "$UNIX_CXX"
-UNIX_CXX=$FINDTOOL_OUT
+#findTool "Cygwin/MSYS G++ compiler" "$UNIX_CXX"
+#UNIX_CXX=$FINDTOOL_OUT
 findTool "MinGW G++ compiler" "$MINGW_CXX"
 MINGW_CXX=$FINDTOOL_OUT
 
-- 
2.8.1

